app/components/ScheduleList/ScheduleList.tsx

```diff
@@ -1,14 +1,14 @@
 import React from 'react';
```

Comment:
If I try to inspect a release schedule, then I get a gray screen.

Reply:
This still happens to me.

Reply:
I fixed it and, and removed the "bigint in state" errors.